.doc_gen/metadata/serverless_metadata.yaml

```diff
@@ -212,6 +212,22 @@ serverless_Kinesis_Lambda_batch_item_failures:
             - description: Reporting Kinesis batch item failures with Lambda using .NET.
               snippet_files:
                 - integration-kinesis-to-lambda-with-batch-item-handling/Function.cs
+    Python:
+      versions:
+        - sdk_version: 3
+          github: https://github.com/aws-samples/serverless-snippets/tree/main/integration-kinesis-to-lambda-with-batch-item-handling
+          excerpts:
+            - description: Reporting Kinesis batch item failures with Lambda using Python.
```

Comment:
These descriptions are Zonbook strings, so use short name entities for any services:
* Kinesis -> &AK;
* Lambda -> &LAM;